exec/cnex/lib/random.c

```diff
@@ -0,0 +1,14 @@
+#include "random.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "cell.h"
+#include "exec.h"
+#include "number.h"
+#include "stack.h"
+
+void random_uint32(TExecutor *exec)
+{
+    push(exec->stack, cell_fromNumber(number_from_uint32(rand())));
```

Comment:
The C `rand()` function returns the same sequence of pseudo-random numbers unless it is seeded with something. Because this is usually unexpected behavior, the random number generated should be seeded with a suitably random value before the first call to `rand()`. At some point we can provide an explicit seed operation to return the same sequence of pseudo-random numbers.